### PR TITLE
Resolve package root correctly

### PR DESCRIPTION
**Pull request by jdb78**
Pyan now searches for __ini__.py files in parents and the infers that this is the package root by default. You can also directly specify a package root.

### CODE REVIEW

**Comment by Technologicat** [comment]
Could you rebase this after we merge in the `black` formatting changes? Would help to see what has changed here. :)

**Comment by jdb78** [comment]
Rebased. Could you have a look again here @Technologicat ?

File: pyan/__init__.py [hunk 1]
```diff
@@ -36,6 +37,7 @@ def create_callgraph(
         filenames: glob pattern or list of glob patterns
             to identify filenames to parse (`**` for multiple directories)
             example: **/*.py for all python files
+        root: path to known root directory at which package root sits. Defaults to None, i.e. it will be infered.
```
Comment (Technologicat) [inline pyan/__init__.py:40]: typo, should be "inferred"

**Review by Technologicat — APPROVED** [review]
LGTM. Spotted just one typo.

File: pyan/__init__.py (hunk 1 above)
Comment (jdb78) [reply pyan/__init__.py]: fixed

**Comment by jdb78** [comment]
@Technologicat , would you like to merge this and make a new release?

**Comment by jdb78** [comment]
@Technologicat just pinging you

**Comment by Technologicat** [comment]
Yes, merging! Sorry, I must have missed the update.

**Pull request merged**